fs/localfs/local_fs_openbsd.go

```diff
@@ -0,0 +1,15 @@
+package localfs
```

Comment:
doesn't `local_fs_nonwindows.go` work here?

Reply:
you probably just need to tweak build flags:

`local_fs_64bit.go` by adding  !openbsd`
`local_fs_32bit.go` by adding `openbsd`

Reply:
Doing that yields : 
```
# github.com/kopia/kopia/fs/localfs
fs/localfs/local_fs_nonwindows.go:26:12: undefined: platformSpecificWidenDev
fs/localfs/local_fs_nonwindows.go:27:13: undefined: platformSpecificWidenDev
```

Reply:
Thank you, it is fixed.